RELEASE.md

```diff
@@ -3,7 +3,11 @@
 The IBM PowerVS Block CSI Driver is released on an as-needed basis. The process is as follows:
 
 1. An issue is proposing a new release with a changelog since the last release
-1. All [OWNERS](OWNERS) must LGTM this release
-1. An OWNER runs `git tag -s $VERSION` and inserts the changelog and pushes the tag with `git push $VERSION`
-1. The release issue is closed
-1. An announcement email is sent to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] ibm-powervs-block-csi-driver $VERSION is released`
\ No newline at end of file
+2. All [OWNERS](OWNERS) must LGTM this release
+3. An OWNER runs `git tag -s $VERSION` and inserts the changelog and pushes the tag with `git push $VERSION`
+4. The release issue is closed
+5. An announcement email is sent to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] ibm-powervs-block-csi-driver $VERSION is released`
+6. A new release(0.x+1.0 where x is the current version) will be cut when there are new releases in the following packages: `kubernetes, go, go builder, container-storage-interface, side cars`
```

Comment:
This can be sampled as
```suggestion
6. A new release will be cut when there are new releases in the following packages: `kubernetes, go, go builder, container-storage-interface, side cars`
```

Reply:
fixed